airwallex/src/main/java/com/airwallex/android/view/AddPaymentMethodActivityLaunch.kt

```diff
@@ -93,6 +100,34 @@ internal class AddPaymentMethodActivityLaunch :
         }
     }
 
+    @Parcelize
+    internal data class CancellationResult internal constructor(
+        val isSinglePaymentMethod: Boolean
+    ) : AirwallexActivityLaunch.Result {
+        override fun toBundle(): Bundle {
+            return Bundle().also {
+                it.putParcelable(AirwallexActivityLaunch.Result.AIRWALLEX_EXTRA, this)
+            }
+        }
+
+        internal companion object : Parceler<CancellationResult> {
```

Comment:
I'm wondering if even without this custom parcelize implementation it also can work.